lib/tasks/temp.rake

```diff
@@ -1,4 +1,39 @@
 namespace :temp do
+  desc 'Migrate PublicBodyCategory into Category model'
+  task migrate_public_body_categories: :environment do
```

Comment:
Probably want to error `if Category.any?` to avoid issues where category already exists

Reply:
Went for `if Category.public_body_tree.any?` as `Category.public_body_root` uses a `find_or_create_by` so if that method is called first (for example loading the authorities list view) then `Category.any?` would be true.